src/components/date-inputs/DateRangeInput.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { FC } from "react";
+// Do not understand why eslint is complaining about this import
+// It is for sure listed in the dependencies of package.json.
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Flatpickr from "react-flatpickr";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import "flatpickr/dist/flatpickr.css";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Danish } from "flatpickr/dist/l10n/da";
+import dayjs from "dayjs";
+
+export interface DateRangeInputProps {
+  setStartDate: (date: string) => void;
+  setEndDate: (date: string) => void;
+  startDate: string;
+  endDate: string;
+  label: string;
+  className?: string;
+  dataCy?: string;
+}
+
+const DateRangeInput: FC<DateRangeInputProps> = ({
+  setStartDate,
+  setEndDate,
+  startDate,
+  endDate,
+  label,
+  className = "date-range",
+  dataCy = "date-range"
+}) => {
+  const refLabel = React.useRef<HTMLLabelElement | null>(null);
+
+  const scrollCalendarIntoView = () => {
+    refLabel.current?.scrollIntoView();
+  };
+
+  // We only create a default date if both start and end date are set
+  // Because it is about defing a range.
+  const value =
+    startDate && endDate
+      ? [dayjs(startDate).toDate(), dayjs(endDate).toDate()]
+      : undefined;
+
+  return (
+    <div data-cy={dataCy} className={className}>
+      <div className="date-range__input">
+        <label
+          ref={refLabel}
+          htmlFor="date-range"
+          className="text-body-medium-regular"
+        >
+          {label}
+        </label>
+        <Flatpickr
```

Comment:
Also, would there be a way to introduce a placeholder text (something like "choose date period", and maybe a little arrow pointing downwards to indicate that this field is interactive? I think it would go a long way.
![image](https://github.com/danskernesdigitalebibliotek/dpl-react/assets/28546954/0418cf87-e6f0-48b7-9b2d-db2b6bf626aa)

Reply:
Great feedback!
I have added a placeholder (also in the cms).
Again I think the time invested in getting it to work with an arrow in the design system is not worth it compared to the time already spent.
If DDF in the future wants an arrow I have made sure a class is added to the field in case it is empty.